ni_measurementlink_service/session_management/_reservation.py

```diff
@@ -199,6 +199,362 @@ def _grpc_channel_pool(self) -> GrpcChannelPool:
         return self._session_management_client._grpc_channel_pool
 
 
+class MultiplexerSessionHandler(_BaseSessionContainer):
+    """Manages multiplexer session information."""
+
+    def __init__(
+        self,
+        session_management_client: SessionManagementClient,
+        multiplexer_session_info: Optional[
+            Sequence[session_management_service_pb2.MultiplexerSessionInformation]
+        ],
+    ) -> None:
+        """Initialize multiplexer object."""
+        super().__init__(session_management_client)
+        self._multiplexer_session_cache: Dict[str, object] = {}
+        self._multiplexer_session_info: Sequence[MultiplexerSessionInformation] = []
+
+        self._multiplexer_session_info = []
+        if multiplexer_session_info is not None:
+            self._multiplexer_session_info = [
```

Comment:
This assigns `self._multiplexer_session_info` three times.

If you rewrite it as if/else, then it will only assign the field once. Also, you probably won't need to explicitly specify a type hint anymore, because type checkers can infer the list element type from the first assignment.

Reply:
Sure, I've updated with an if/else.